LiberationOfUndermine/Locales/frFR.lua

```diff
@@ -1,98 +1,98 @@
+	L.under_you_comment = "E" -- Implies this setting is for the damage from the ground effect under you
 end
 
 L = BigWigs:NewBossLocale("The One-Armed Bandit", "frFR")
 if L then
-	--L.rewards = "Prizes" -- Fabulous Prizes
-	--L.rewards_desc = "When two Tokens are locked in, the \"Fabulous Prize\" is dispensed.\nMessages will let you know which one has been rewarded.\nThe infobox will show which prizes are still available."
-	--L.deposit_time = "Deposit Time" -- Timer that indicates how long you have left to deposit the tokens.
+	L.rewards = "Prix" -- Fabulous Prizes
+	L.rewards_desc = "Lorsque deux jetons sont verrouillés, un  \"prix fabuleux\" est distribué.\nLes messages vous laisseront savoir lequel a été obtenu.\nLa boîte d'info affichera quels prix sont encore disponibles."
+	L.deposit_time = "Temps de dépôt" -- Timer that indicates how long you have left to deposit the tokens.
 
-	--L.pay_line = "Coins"
-	--L.shock = "Shock"
-	--L.flame = "Flame"
-	--L.coin = "Coin"
+	L.pay_line = "Ligne pièces"
+	L.shock = "Champ"
+	L.flame = "Flamme"
+	L.coin = "Pièce"
 end
 
 L = BigWigs:NewBossLocale("Mug'Zee, Heads of Security", "frFR")
 if L then
-	--L.earthshaker_gaol = "Prisons"
-	--L.frostshatter_boots = "Frost Boots" -- Short for Frostshatter Boots
-	--L.frostshatter_spear = "Frost Spears" -- Short for Frostshatter Spears
-	--L.stormfury_finger_gun = "Finger Gun" -- Short for Stormfury Finger Gun
-	--L.molten_gold_knuckles = "Tank Frontal"
-	--L.unstable_crawler_mines = "Mines"
-	--L.goblin_guided_rocket = "Rocket"
-	--L.double_whammy_shot = "Tank Soak"
-	--L.electro_shocker = "Shocker"
+	L.earthshaker_gaol = "Prisons"
+	L.frostshatter_boots = "Bottes de givre" -- Short for Frostshatter Boots
+	L.frostshatter_spear = "Lances de givre" -- Short for Frostshatter Spears
+	L.stormfury_finger_gun = "Doigt-pistolet" -- Short for Stormfury Finger Gun
+	L.molten_gold_knuckles = "Frontal Tank"
+	L.unstable_crawler_mines = "Mines"
+	L.goblin_guided_rocket = "Fusée"
+	L.double_whammy_shot = "Tank Soak"
+	L.electro_shocker = "Électrochoqueur mod. II"
 end
 
 L = BigWigs:NewBossLocale("Chrome King Gallywix", "frFR")
 if L then
-	--L.story_phase_trigger = "What, you think you won?" -- What, you think you won? Nah, I got somethin' else for ya.
+	L.story_phase_trigger = "Vous pensez avoir gagné ?" -- What, you think you won? Nah, I got somethin' else for ya.
 
-	--L.scatterblast_canisters = "Cone Soak"
-	--L.fused_canisters = "Group Soaks"
-	--L.tick_tock_canisters = "Soaks"
-	--L.total_destruction = "DESTRUCTION!"
+	L.scatterblast_canisters = "Cone Soak"
+	L.fused_canisters = "Groupe Soaks"
+	L.tick_tock_canisters = "Soaks"
+	L.total_destruction = "DESTRUCTION !"
 
-	--L.duds = "Duds" -- Short for 1500-Pound "Dud"
-	--L.all_duds_detontated = "All Duds Detonated!"
-	--L.duds_remaining = "%d |4Dud remains:Duds remaining;" -- 1 Dud Remains | 2 Duds Remaining
```

Comment:
Please do not remove the code needed to distinguish between singular and plural.